PySDM/physics/constants_defaults.py

```diff
@@ -108,6 +108,10 @@
 k_l19_b = 1.0456 * si.dimensionless
 k_l19_c = 0.017 / si.kelvin
 
+# Delta v for diffusivity
+l19_dv = 0.0 * si.metres
```

Comment:
BTW, we can likely change the "l19" prefix to something related to P&K or Okuyama and Zung?